src/com/hms_networks/americas/sc/thingworx/TWConnectorMain.java

```diff
@@ -216,7 +216,12 @@ public static void main(String[] args) {
         "Starting " + TWConnectorConsts.CONNECTOR_NAME + " " + TWConnectorConsts.CONNECTOR_VERSION);
 
     // Calculate local time offset and configure queue
-    TWTimeOffsetCalculator.calculateTimeOffsetMilliseconds();
+    try {
+      TWTimeOffsetCalculator.calculateTimeOffsetMilliseconds(
+          connectorConfig.getFtpUser(), connectorConfig.getFtpPassword());
+    } catch (JSONException e3) { // TODO Auto-generated catch block
+      e3.printStackTrace();
```

Comment:
I missed this one. TODO should be removed and exception/log should use logger

Reply:
Fixed. I think that was the last one